src/modules/account/const/index.tsx

```diff
@@ -102,3 +104,32 @@ export enum Permission {
 	EDIT_OWN_MATERIAL_REQUESTS = 'EDIT_OWN_MATERIAL_REQUESTS',
 	DELETE_OWN_MATERIAL_REQUESTS = 'DELETE_OWN_MATERIAL_REQUESTS',
 }
+
+export enum Group {
+	VISITOR = 'VISITOR',
```

Comment:
Q: zeker dat deze enum niet al ergens bestaat 🤔 Zou ik vreemd vinden als het niet zo is maar als het zo is mag je deze negeren.

Reply:
No enum like this in de codebase.